nengo/utils/progress.py

```diff
@@ -212,12 +236,197 @@ def _get_in_progress_line(self, progress):
 
         return '\r' + line.format(progress_str)
 
+    def _get_unknown_progress_line(self, progress):
+        duration = progress.elapsed_seconds()
```

Comment:
Maybe add a comment here that says it animates a marker continuously to show progress? It took me a while to understand what the code was doing without that context.